tests/test_database.py

```diff
@@ -3414,18 +3411,29 @@ def test_multiplex_dump(self):
 
     def test_multiplex_sym_dump(self):
         db = cantools.db.load_file('tests/files/sym/test_multiplex_dump.sym')
+        # change the name of the multiplexer signal to empty to trigger the condition in function _dump_message
+        db.messages[1].signals[0].name = ''
         dumped_db = cantools.db.load_string(db.as_sym_string())
-        dumped_msg = dumped_db.get_message_by_frame_id(0x100)
 
-        # Note: cantools database cannot support multiple multiplexer signal names, so SYM file names the multiplexer
-        # signal after the multiplexer id (Hence, 2A, not MultiplexerSig)
```

Comment:
@samc1213: I don't get that comment. Do you remember what you meant to say here? (In my understanding, assigning more than one multiplexer selector signal to any multiplexed signal does not make sense anyway?)

Reply:
[The relevant pull request.](https://github.com/cantools/cantools/pull/431).

Reply:
@andlaus Are there any updates for this? Is there anything we can do to help approve this pull request?

Reply:
I am just confused why this comment was made in the first place, but I think that it was spurious. (so everything is alright here as far as I can see...) I'll merge the PR once the special treatments for empty signal names (and the corresponding test) are removed...

Reply:
The files are already updated. Please let me know if there are any other issues. We really hope that this feature can be merged and it's an important feature for our product implementation. Thanks @andlaus

Reply:
I'm sorry for the late reply. I can't recall why I thought SYM multiplexer and cantools database representations didn't match, and decided to use the multiplexer value as the signal name. I thought during parsing we would lose the name of each multiplexer, so we had to use the value or something. I tried to reproduce but couldn't.